### ISSUE TITLE
mark foreign words with lang attribute

### ISSUE THREAD

**Issue by vellip** [liqd/a4-meinberlin#520]
Words like:
* Feedback

should be marked with a `<span lang="en">text</span>` or similar.

**Comment by xi** [comment]
"Feedback" seems to be a german word: http://www.duden.de/rechtschreibung/Feedback

Are there any words that are not translated at all?

**Comment by vellip** [comment]
BITV says to those:
> Die Wortlisten von Screenreadern sollten auch geläufige Fremdwörter enthalten. Dies ist jedoch nicht durchgängig der Fall, auch geläufige Fremdwörter werden ohne Auszeichnung falsch ausgesprochen. Daher macht ihre Auszeichnung Sinn, auch wenn sie nicht gefordert wird.

Not translated should not happen, except it's a bug (like accepting ToS).
